kubernetes/gke-base/cfg/values.yaml

```diff
@@ -0,0 +1,87 @@
+confluentPlatformImage: &cpImage
+  image: 
+    tag: 5.3.0.0
+ 
+global:
+  initContainer:
+    <<: *cpImage
+  provider:
+    registry:
+      fqdn: docker.io 
+    name: gcp 
+    storage:
+      provisioner: kubernetes.io/gce-pd 
+      reclaimPolicy: Delete 
+      parameters:
+        type: pd-ssd 
+    sasl:
+      plain:
+        username=test
+        password=test123
+
+manager:
+  name: manager
+
+operator:
+  name: operator
+  image:
+    tag: 0.176.1
+
+zookeeper:
+  <<: *cpImage
+  resources:
+    cpu: 200m
+    memory: 512Mi
+  
+kafka:
+  <<: *cpImage
+  resources:
+    cpu: 200m
+    memory: 1Gi
+  loadBalancer:
+    enabled: false
+  tls:
+    enabled: false
+  metricReporter:
+    enabled: true
+  configOverrides:
+    server:
+    - "auto.create.topics.enabled=true"
+
+schemaregistry:
+  <<: *cpImage
+  dependencies:
+    kafka:
+      bootstrapEndpoint: kafka:9071 
+
+connect:
+  image:
+    repository: cnfldemos/cp-server-connect-operator-with-datagen 
+    tag: 5.3.0.0_0.1.3
+  tls:
+    enabled: false
+  loadBalancer:
+    enabled: false
+  dependencies:
+    kafka:
+      brokerCount: 1
+      bootstrapEndpoint: kafka:9071
+    schemaregistry:
+      enabled: true 
+      url: http://schemaregistry:8081 
+
+controlcenter:
```

Comment:
@rspurgeon I'm surprised these aren't already the default set for C3...

Reply:
If I remove these (SR also), I get helm rendering errors.  I can bring it up to engineering, but will leave in for now.

Reply:
Definitely leave it in for now, separately we can discuss with engineering